post_office/apps.py

```diff
@@ -10,5 +10,4 @@ def ready(self):
         from post_office import tasks
         from post_office.signals import email_queued
 
-        if hasattr(tasks, 'queued_mail_handler'):
-            email_queued.connect(tasks.queued_mail_handler)
+        email_queued.connect(tasks.queued_mail_handler)
```

Comment:
I think this signal should only be connected if `CELERY_ENABLED` is `True`.

Reply:
I'm not a friend of duck typing. The proposed solution uses a non-operational placeholder function [queued_mail_handler](https://github.com/ui/django-post_office/blob/78a15fa4d6b08cbeb0327c161d7d5533af8de682/post_office/tasks.py#L22) in case Celery is missing. This in my opinion is a more robust solution. Consider, if other queuing frameworks, such as for example [django-q](https://django-q.readthedocs.io/en/latest/) would be integrated into **Post-Office**.

Reply:
Checking whether `CELERY_ENABLED` settings is `True` is not duck typing.

If Celery is not enabled, there's no need to connect to this signal. If I remember correctly, Django has an optimization that if there's nothing connected to a signal, the signal won't be sent.

Reply:
> If Celery is not enabled, there's no need to connect to this signal. If I remember correctly, Django has an optimization that if there's nothing connected to a signal, the signal won't be sent.

This additional condition might safe a few milliseconds at the cost of more complexity. Anyway, I added this condition.